code/modules/mob/living/carbon/human/life/handle_environment.dm

```diff
@@ -96,3 +83,31 @@
 		return HEAT_DAMAGE_LEVEL_3
 	else
 		return 0
+
+/mob/living/carbon/human/proc/get_thermal_loss(var/datum/gas_mixture/environment)
+	var/loc_temp = get_loc_temp(environment)
+	if(loc_temp < bodytemperature)
+		// We're going to try and just use exposed area(temperature difference)/cold divisor, and assume we're only conducting.
+		var/thermal_loss = (1-get_cold_protection())  				// How much of your skin is exposed.
+		thermal_loss	*= environment.total_moles/103.934			// Multiplied by how many moles are in the environment over 103.934, the normal value of a station - this means space will not take heat from you.
+		thermal_loss	*= (get_skin_temperature() - loc_temp)		// Multiplied by the difference between you and the room temperature
+		thermal_loss	/= BODYTEMP_COLD_DIVISOR					// Divided by the cold_divisor
+		world << "[thermal_loss]"
```

Comment:
It's a good thing I test my code 100%*!

*on expected differences resulting from change(s) only